WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -179,6 +189,26 @@ public void onErrorResponse(VolleyError error) {
                 });
     }
 
+    private void fetchJetpackSettings() {
+        WordPress.getRestClientUtils().getJetpackSettings(
+                mSite.getSiteId(), new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        AppLog.d(AppLog.T.API, "Received response to Jetpack Settings REST request.");
```

Comment:
I generally lean `w` if the error is handled gracefully. `e` probably makes more sense here.